deploy/helm/node-collector-config.yaml

```diff
@@ -452,11 +452,30 @@ receivers:
                 static_configs:
                   - targets:
                       - '`endpoint`:`"prometheus.io/port" in annotations ? annotations["prometheus.io/port"] : 9090`'
+
+      {{- if .Values.otel.metrics.autodiscovery.prometheusEndpoints.podMonitors }}
+      {{- range $key, $rule :=.Values.otel.metrics.autodiscovery.prometheusEndpoints.podMonitors.rules }}
+      prometheus/discovery/{{ $key }}:
+        rule: {{ $rule.rule }}
+        config:
+          config:
+            scrape_configs:
+              - job_name: pod
+                scheme: "http"
+                scrape_interval: {{ $rule.scrape_interval | default "60s" | quote}}
```

Comment:
instead 60s would be nice to use .Values.otel.metrics.prometheus.scrape_interval

Reply:
I tried, but you cant use global inside loops.